src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/CrossTarget/CrossTargetSubscriptionHostBase.cs

```diff
@@ -78,7 +78,7 @@ internal abstract class CrossTargetSubscriptionHostBase : OnceInitializedOnceDis
 
         protected async Task AddInitialSubscriptionsAsync()
         {
-            using (_tasksService.LoadedProject())
```

Comment:
So previously the object returned by `LoadedProject` would have been disposed at the end of the block. Do we need to do something similar to the object returned by `LoadedProjectAsync`?